addons/common/XEH_postInit.sqf

```diff
@@ -55,6 +56,21 @@
     _vehicle setVariable [QGVAR(blockEngine), _set > 0, true];
     _vehicle engineOn false;
 }] call CBA_fnc_addEventHandler;
+[QGVAR(blockThrow), {
+    params ["_object", "_set"];
+    TRACE_2("blockThrow EH",_object,_set);
```

Comment:
I really don't like messing with the setting var directly

I would move this to advThrowing with
```
GVAR(allowed) = true;
[QEGVAR(common,blockThrow), {
    params ["_object", "_set"];
    TRACE_2("blockThrow EH",_object,_set);

    GVAR(allowed) = (_set <= 0);
}] call CBA_fnc_addEventHandler;
```

and then in canPrepare.sqf check `GVAR(enabled) && GVAR(allowed)`

Reply:
even better, just check `(ace_player getVariable ["ace_common_effect_blockthrow", 0]) <= 0` in can prepare